pkg/registry/networking/ingress/strategy_test.go

```diff
@@ -69,6 +71,11 @@ func newIngress() networking.Ingress {
 
 func TestIngressStrategy(t *testing.T) {
 	ctx := genericapirequest.NewDefaultContext()
+	apiRequest := genericapirequest.RequestInfo{APIGroup: "networking.k8s.io",
+		APIVersion: "v1beta1",
+		Resource:   "ingress",
```

Comment:
should be `ingresses`, right?

Reply:
Fixed, thanks! Not sure how it worked before?